test/language/expressions/object/property-name-bigint-literals.js

```diff
@@ -0,0 +1,13 @@
+// Copyright (C) 2013 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+  description: >
+      BigInt literals may be used as a literal property name in an object
+      literal.
+  es6id: 12.2.6
+---*/
```

Comment:
I think that this test needs `features` field.

    features: [BigInt, destructuring-binding]

Reply:
Thanks! Updated.